recipe/meta.yaml

```diff
@@ -11,6 +11,8 @@ package:
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ checksum }}
+  patches:
+    {% if version == "8.7.0" %}- 0001-restore-pygments-lexers.patch{% endif %}
```

Comment:
isn't this solvable by a simple selector [version == "8.7.0"] ?

Reply:
and btw "patches:" should have the selector too

Reply:
I've decided to add a comment without jinja2 templates or selectors:
```
  # Restoring pygments lexers is needed only for v8.7.0. 
  # Check if it can be removed in the next release.
  patches:
    - 0001-restore-pygments-lexers.patch
```